tools/defaults.bzl

```diff
@@ -249,7 +250,7 @@ def ng_web_test_suite(deps = [], static_css = [], bootstrap = [], tags = [], **k
             # Note: when changing the browser names here, also update the "yarn test"
             # script to reflect the new browser names.
             "@io_bazel_rules_webtesting//browsers:chromium-local",
-            "@io_bazel_rules_webtesting//browsers:firefox-local",
+            # "@io_bazel_rules_webtesting//browsers:firefox-local",
```

Comment:
What's the reason for this?

Reply:
oops, i'll fix this one, was just for local testing, for some reason i can't get headless firefox to run properly on my mac